Hai Reveal intro: rewrite pug media coverage with less narrator editorializing, as several missions in a new file (#7760)
endless-sky/endless-sky: data/human/war reactions.txt

# this program. If not, see <https://www.gnu.org/licenses/>.

########################################################################

# This file contains human reactions to the pug war, and events
# leading up to it.  It lets us know how regular, everyday people felt
# about the war. This leads to the reveal of the Hai, who are unable
# to stay hidden with so many people thinking about aliens.

mission "00 Post-War Reactions Begin"
	landing
	invisible
	to offer
		has "main plot completed"
	on offer
		event "post-war reactions begin" 20 30
		fail

event "post-war reactions begin"

mission "War Reactions: Asteroid Miners"
	landing
	minor
	source
		government "Republic" "Syndicate" "Free Worlds"
		# These miners were far from the war the whole time.
		attributes "deep" "paradise" "north"
	to offer
		has "event: post-war reactions begin"
		random < 30
	on offer
		set "pug media coverage"
		log "Minor People" "Maduena Kabiru" `A senator from Bounty in the Delta Velorum system. He is somewhat paranoid, and insists upon the Navy taking greater steps to better explore the galaxy beyond human space to search for possible threats.`
		conversation

> 			`As you leave your ship, you see a crowd watching a video on a interactive information display, not far from your landing spot. The video jumps between a panel discussion and footage of Admiral Danforth being quizzed by Parliament. It finishes before you arrive, and many of the onlookers have left for their business, but two captains remain. By the sounds of it, they're having a passionate discussion about the video.`

Reviewer: ```suggestion
			`As you leave your ship, you see a crowd watching a video on a interactive information display not far from your landing spot. The video jumps between a panel discussion and footage of Admiral Danforth being quizzed by Parliament. It finishes before you arrive, and many of the onlookers have left for their business, but two captains remain. By the sounds of it, they're having a passionate discussion about the video.`
```
This comma isn't needed. I'm glad it's not one I suggested.

				`	"It was good meeting you both, but I must get back to my business now."`
					decline
			label aliens
			action
				set "War Reactions: Navy and Free Worlds Captains: label aliens"
			`	"Only the usual assortment of hoaxes," says Rain. The sisters both laugh at this.`
			`	"Seriously, though, both the Free Worlds Senate and Republic Parliament are up in arms over fear of aliens," says Fiona. Rain adds, "Only metaphorically up in arms, this time.`
			`	"If we find new aliens, we'll let the necessary authorities know as soon as possible. You should do the same. Be discrete, avoid conflicts, and seek help if you need it."`
				goto questions
			label navy
			action
				set "War Reactions: Navy and Free Worlds Captains: label navy"
			`	"Joint police operation," says Captain Rain. "It's more common than you may think."`
			`	Captain Fiona nods. "We're shutting down a galactic Spider Funk smuggling operation. It's an illegal hallucinogenic. Not addictive, but still dangerous. The organization running it is huge, well funded, and well armed."`
			`	"Not a match for a Dreadnought and an Auxiliary, though," brags Rain.`

> 			`	"We have to fly all over Republic and Free Worlds space, so we have a joint patrol with one ship from each country."`

Reviewer: ```suggestion
			`	"We have to fly all over Republic and Free Worlds space, so we have a joint patrol with one ship from each state."`
```
I've never seen 'country' used this way in Endless Sky and I have a feeling that, after a thousand years, it's not used this way anymore, except on Thule.

			`	"Of course, we have another mission," says Fiona. "Everywhere we go, we look out for new alien activity."`
				goto questions
			label sister
			action
				set "War Reactions: Navy and Free Worlds Captains: label sister"

> 			`	"Yup," Fiona says. "I blew up her Cruiser! When her ship's escape pods were veering towards a star, we rescued them."`
> 			`	Rain continues. "Can you imagine my surprise when I woke up on my sister's new Dreadnought?"`

Reviewer: ```suggestion
			`	"Yup," Fiona says, "I blew up her Cruiser! When her ship's escape pods were veering towards a star, we rescued them."`
			`	Rain continues, "Can you imagine my surprise when I woke up on my sister's new Dreadnought?"`
```
We've changed commas in these situations so many times that you get it wrong in the only places where commas should be stuck!

> 			`	"Hey, you got a new Auxiliary commission afterwards," replies Fiona. "and your whole crew survived. Now we escort each other's ships during investigations."`

Reviewer: ```suggestion
			`	"Hey, you got a new Auxiliary commission afterwards," replies Fiona, poking Rain as she says it, "and your whole crew survived. Now we escort each other's ships during investigations!"`
```
Just feel like a poke sounds about right here.

				goto questions
			label mercenary
			action
				set "War Reactions: Navy and Free Worlds Captains: label mercenary"

> 			`	"I meant no offense," says Fiona. "You're quite the hero." She thinks for a second and looks at her sister. "Rain, can you explain?"`

Reviewer: ```suggestion
			`	"I meant no offense," says Fiona, "you're quite the hero." She thinks for a second and glances at her sister. "Rain, can you explain?"`
```

> 			`	Rain says, "We're uniformed career officers who must follow orders. Our country owns the ships, uniforms, and everything else we use. Our salaries may be meager compared to your earnings, but serving one's country is a rich reward.`

Reviewer: ```suggestion
			`	Rain says, "We're uniformed career officers who must follow orders. Our states own the ships, uniforms, and everything else we use. Our salaries may be meager compared to your earnings, but serving one's state is a rich reward.`
```

> # Copyright (c) 2022 UnorderedSigh and MasterOfGrey

Reviewer 2: ```suggestion
# Copyright (c) 2022 Michael Zahniser
```

# this program. If not, see <https://www.gnu.org/licenses/>.

########################################################################

# This file contains human reactions to the pug war, and events
# leading up to it.  It lets us know how regular, everyday people felt
# about the war. This leads to the reveal of the Hai, who are unable
# to stay hidden with so many people thinking about aliens.

mission "00 Post-War Reactions Begin"
	landing
	invisible
	to offer
		has "main plot completed"
	on offer
		event "post-war reactions begin" 20 30
		fail

event "post-war reactions begin"

mission "War Reactions: Asteroid Miners"
	landing
	minor
	source
		government "Republic" "Syndicate" "Free Worlds"
		# These miners were far from the war the whole time.
		attributes "deep" "paradise" "north"
	to offer
		has "event: post-war reactions begin"
		random < 30
	on offer
		set "pug media coverage"
		log "Minor People" "Maduena Kabiru" `A senator from Bounty in the Delta Velorum system. He is somewhat paranoid, and insists upon the Navy taking greater steps to better explore the galaxy beyond human space to search for possible threats.`
		conversation

> 			`As you leave your ship, you see a crowd watching a video on a interactive information display not far from your landing spot. The video jumps between a panel discussion and footage of Admiral Danforth being quizzed by Parliament. It finishes before you arrive, and many of the onlookers have left for their business, but two captains remain. By the sounds of it, they're having a passionate discussion about the video.`

Author: You  had two conflicting suggestions and I used the other one.

			choice
				`	"Hey! What's this video about?"`
				`	(Leave them be.)`
					decline
			`	You're startled when the two captains turn to you, twin men with preposterous sideburns and rat-like mustaches. It's impossible to tell if they're smiling or frowning, as the pair live squarely in the middle of the uncanny valley.`
			`	The left twin asks, "Can you believe this? It's been weeks and everyone's still obsessing over the Pug invasion. The news was all experts and officials at first, but now we just see talking heads and conspiracy theorists."`
			`	In disbelief, the right twin responds, "It's just a propaganda exercise to boost Syndicate sales. Even if these elusive 'Pug' existed, we won't hear the end of it as long as the story boosts public opinion of the Syndicate."`
			choice
				`	"It was possibly the biggest event this century. Of course people are going to keep talking about it."`

> 				`	"Don't you see it was what we needed? The Pug caused us to set aside our pride and resolve our issues directly without continuing the conflict."`

Reviewer 3: ```suggestion
				`	"Don't you see it was what we needed? The Pug made us set aside our pride and resolve our issues directly without continuing the conflict."`
```

			`	The right one says apologetically, "Thank you for your service, Captain <last>."`
			`	They both give clumsy but well-intentioned salutes and return to their ships.`
				decline
			label eyes
			`	The left twin laughs and says, "Pardon us if we keep our eyes open and trust to our own guns." Both twins nod and depart to their ships.`
				decline
			label shrug
			`	Both twins shrug and leave for their ships.`
				decline
			label scowl
			`	Both twins' faces twist in a way that might indicate embarrassment or annoyance. It's hard to tell since their facial hair is halting any emotive power. They walk away in the direction of their ships.`
				decline

mission "War Reactions: Navy and Free Worlds Captains"
	landing
	minor
	source
		government "Republic" "Syndicate" "Free Worlds"
		attributes "spaceport"
		# This distance from Sirius puts them a few jumps from the edge of human space, or anywhere in the Free Worlds
		near "Sirius" 5 100
	to offer
		has "event: post-war reactions begin"
		random < 30
	on offer
		set "pug media coverage"
		conversation

> 			`You are relaxing in a cafe while your ship refuels, and a pair of middle-aged captains come to your table. One is Navy, the other is Free Worlds, and they're obviously sisters.`

Reviewer 3: ```suggestion
			`You are relaxing in a cafe while your ship refuels, and a pair of middle-aged captains come to your table. One is Navy, the other is Free Worlds, and they bear a striking resemblance to each other.`
```

			`	"Captain Rain, R.N.S. Camuy," says one. "How's the <ship>, Captain <last>?"`

> 			`	"Captain Fiona, F.S. Baobab," says her sister. "We both saw you in battle. You're the war's most accomplished mercenary!`

Reviewer 3: ```suggestion
			`	"Captain Fiona, F.S. Baobab," says the other. "We both saw you in battle. You're the war's most accomplished mercenary!`
```

			`	"We've been patrolling together and we saw your ship, so we've come to say 'hi.'"`
			label questions
			choice
				`	"Why are Navy and Free Worlds captains traveling together?"`
					goto navy
					to display
						not "War Reactions: Navy and Free Worlds Captains: label navy"
				`	"Have you found any new aliens?"`
					goto aliens
					to display
						not "War Reactions: Navy and Free Worlds Captains: label aliens"
						has "War Reactions: Navy and Free Worlds Captains: label navy"

> 				`	"Did you fight on the other side of the war from your sister?"`

Reviewer 3: ```suggestion
				`	"Did you fight on the other side of the war from each other?"`
```

					goto sister
					to display

> 						not "War Reactions: Navy and Free Worlds Captains: label sister"
> 				`	"Mercenary? I'm not a mercenary."`

Reviewer 3: ```suggestion
						not "War Reactions: Navy and Free Worlds Captains: label sister"
				`	"You were on opposite sides as Sisters!?"`
					goto sister2
					to display
						has "War Reactions: Navy and Free Worlds Captains: label sister"
						not "War Reactions: Navy and Free Worlds Captains: label sister2"
				`	"Mercenary? I'm not a mercenary."`
```

Reviewer: Should 'Sisters' be capitalised?

Reviewer 3: Indeed, not, the copypasta monster came and did it. I've fixed it.

			`	"If we find new aliens, we'll let the necessary authorities know as soon as possible. You should do the same. Be discrete, avoid conflicts, and seek help if you need it."`
				goto questions
			label navy
			action
				set "War Reactions: Navy and Free Worlds Captains: label navy"
			`	"Joint police operation," says Captain Rain. "It's more common than you may think."`
			`	Captain Fiona nods. "We're shutting down a galactic Spider Funk smuggling operation. It's an illegal hallucinogenic. Not addictive, but still dangerous. The organization running it is huge, well funded, and well armed."`
			`	"Not a match for a Dreadnought and an Auxiliary, though," brags Rain.`
			`	"We have to fly all over Republic and Free Worlds space, so we have a joint patrol with one ship from each state."`
			`	"Of course, we have another mission," says Fiona. "Everywhere we go, we look out for new alien activity."`
				goto questions
			label sister
			action
				set "War Reactions: Navy and Free Worlds Captains: label sister"

> 			`	"Yup," Fiona says, "I blew up her Cruiser! When her ship's escape pods were veering towards a star, we rescued them."`

Reviewer 3: ```suggestion
			`	"Yup," Fiona says, "being my sister didn't stop me blowing up her Cruiser! We rescued her ship's escape pods though, they were veering concerningly towards a star."`
```

Reviewer: ```suggestion
			`	"Yup," Fiona says, "being my sister didn't stop me blowing up her Cruiser! We rescued her ship's escape pods, but only when they began to veer dangerously towards a star."`
```
'veering concerningly' could be improved?

Reviewer 3: In this case I don't think so for speech, or at least not in this way. You added like 6 extra syllables and made it sound stilted.

Reviewer 4: ```suggestion
			`	"Yup," Fiona says, "being my sister didn't stop me blowing up her Cruiser! We rescued her ship's escape pods, though; they were veering concerningly towards a star."`
```

Fix that run on, at least.

I do agree that "veering concerningly towards a star" is some awkward phrasing, but I'm getting a bit of awkward vibe from these sisters, anyway. It seems well enough within character as dialogue.

			`	Rain continues, "Can you imagine my surprise when I woke up on my sister's new Dreadnought?"`
			`	"Hey, you got a new Auxiliary commission afterwards," replies Fiona, poking Rain as she says it, "and your whole crew survived. Now we escort each other's ships during investigations!"`

> 				goto questions
> 			label mercenary

Reviewer 3: ```suggestion
				goto questions
			label sister2
			action
				set "War Reactions: Navy and Free Worlds Captains: label sister2"
			`	They share a glance and Fiona shrugs. "Sometimes that's just what happens with family."`
			`	Rain continues, "You do what you have to to follow your orders and look out for each other in between."`
				goto questions
			label mercenary
```

			action
				set "War Reactions: Navy and Free Worlds Captains: label mercenary"
			`	"I meant no offense," says Fiona, "you're quite the hero." She thinks for a second and glances at her sister. "Rain, can you explain?"`
			`	Rain says, "We're uniformed career officers who must follow orders. Our states own the ships, uniforms, and everything else we use. Our salaries may be meager compared to your earnings, but serving one's state is a rich reward.`

> 			`	"You, on the other hand, own your ship and serve whoever you want. To keep up with costs, you have to charge a lot of money for your services. Mercenaries only survive as long as you if they're skilled at both finances and combat."`

Reviewer 4: I feel like this could use an explicit distinction in regards to loyalty not undermining their argument. Something like (but not necessarily):

```suggestion
			`	"You, on the other hand, own your ship and serve whoever you want. Even if you remain loyal to a group or cause, how and when you act is still at your own discretion. As well, to keep up with the costs of that independent ownership, you have to charge a lot of money for your services. Mercenaries only survive as long as you if they're skilled at both finances and combat."`
```

				goto questions

mission "War Reactions: Syndicate Orphan"
	landing
	minor
	source
		government "Syndicate"
		not attributes "station"
	to offer
		has "event: post-war reactions begin"
		random < 30
	on offer
		set "pug media coverage"
		conversation

> 			`	When you exit your ship, you hear sounds of fists falling on skin and a kid yelling for help. Around the corner are three boys hitting a fourth, who just fell on the ground. It's not clear if they're going to stop attacking him.`

Reviewer 3: This situation is nonsensical near to any kind of managed landing zone.

Reviewer 4: I'm gonna disagree with that. I've seen this and worse among older children/teens, both as an adult recently and while a child myself, even in well maintained/regulated areas and even in the direct presence of adults, many of whom did exactly `(Ignore them and keep walking.)`

Author: Judging by the missions I've read, the Syndicate spaceports are rife with smuggling and bribery. It's perfectly reasonable for kids to be able to sneak into places they shouldn't be. As long as the kids aren't messing with the spaceships, many adults won't care. This may seem out-of-place in a shiny first-world country, but in other places, the rules aren't quite so simple.

Author: The mission now begins with, "When you exit the secure landing area..."

Reviewer 3: That seems fine. To a degree I imagine a spaceport of any meaningful size is more like an airport. You have to actually disembark *into* the spaceport. Not merely step off your ship and "walk around a corner".
Of course once you're *in* the spaceport yeah I don't see a lot of issue with stuff like that happening there.

Author: This can happen at any Syndicate planet, so I'm trying to be vague about the relative positions of things.

			choice
				`	"Break it up, kids."`
				`	(Ignore them and keep walking.)`
					decline
			`	The three attackers turn around, looking furious. They eye your gun and run away. "Thanks. I'm Gus," says the fourth boy, as he slowly rises to his feet. He looks about twelve years old.`
			`	It takes Gus a few seconds to recover. He looks worried and asks, "Can you walk me to the factory? I'm already late for work, and I'm afraid they'll attack me again."`
			label questions
			choice
				`	"Why were those boys attacking you?"`
					goto attacking
					to display
						not "War Reactions: Syndicate Orphan: attacking"
				`	"Aren't you a little young to work in a factory?"`
					goto young
				`	"I'm in a hurry. You're on your own."`
					decline
			label attacking
			action
				set "War Reactions: Syndicate Orphan: attacking"

> 			`	"My sister is a Private, Third Class, in the Syndicate's Ancillary Security Service, which monitors Syndicate-owned workplaces. Those boys are factory workers, and Syndicate security officers aren't nice to factory workers. They can't attack my sister, so they attack me instead."`

Reviewer 3: The syndicate doesn't sound like the sort of place that would bind his sister's hands from taking retributive action against these boys, so the justification feels weak.

Reviewer 4: This bit I agree with, though, because even though such weak, ultimately self destructive justifications are indeed often used in real life, to me, this is an instance where fiction must be more sensible in order to not feel strained. I'd suggest coming up with a specific incident of her coming down on their colleagues (as example) for a more direct motivation to lash out at him as a surrogate.

Author: The boys can't attack his sister because his sister is part of the Syndicate's security forces. Attacking security officers has rather severe consequences. The mission does not address whether the sister takes revenge against the boys since that isn't relevant to the mission (it would happen afterwards).

Author: The justification has been improved. I misunderstood your comments in my prior PR.

				goto questions
			label young
			`	As you start walking, he answers. "No, I'm an orphan." He sees your confusion and explains. "The Syndicate lets some orphans work in factories, in return for better education, good housing, and medical care. Only the cleverest of us get to do it. All of the orphanage factories are safe; we don't use big dangerous equipment. Just ordinary stuff, like hammers, soldering irons, or sewing machines.`
			`	"Wait, you're a captain, right? Were you in the big fights? I was on Maker when I saw some Pug ships fighting human ships on our Syndicate school's big educational viewscreen."`
			choice
				`	"Yes, I'm Captain <first> <last>."`
					goto me
				`	"I blew up my share of Pug ships."`
					goto blew
				`	"Actually, I led the battle."`

> 			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You're Captain <first> <last>! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand and you shake it.`

Reviewer 4: ```suggestion
			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You're Captain <first> <last>! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
```

				goto next
			label me

> 			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You led the attack! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand and you shake it.`

Reviewer 4: ```suggestion
			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You led the attack! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
```

				goto next
			label blew
			`	"I bet you did! I wish I was on one of our warships, bravely defending the human race. The honor, the glory, the mercenary pay. It sounds amazing."`
			label next
			`	"What was it like, fighting the evil Pug?"`

			choice
				`	"Actually, they're not evil. Their 'war' was a ploy to get us to stop fighting among ourselves. A lot of Pug gave their lives willingly to save us from self-destruction."`
				`	"War is hell, kid. I'm not going to glorify it for you."`
					goto hell
				`	"Leading a battle is like chess with guns and engines. It's the most exhilarating feeling I've ever felt."`
					goto chess

> 				`	"I was an honor to protect the human race from our attackers."`

Reviewer: ```suggestion
				`	"It was an honor to protect the human race from our attackers."`
```
If it survives MasterOfGrey's review...

					goto honor

> 			`	"That's so insightful! Still, without people like you, the Pug's ploy would never have worked. Thank you for your service, captain."`

Reviewer 4: ```suggestion
			`	"That's so insightful! Still, without people like you, the Pug's ploy would never have worked. Thank you for your service, Captain."`
```

				goto salute
			label hell
			`	"You went through hell for us, eh? That's something to think about. Maybe I should stick to engineering instead of becoming a combat pilot. I dunno."`
				goto salute
			label chess
			`	"The Syndicate is training me to maintain and pilot combat ships! One day, I'll fight aliens like you did."`
				goto salute
			label honor
			`	"Glory, honor, and service. The Syndicate has been training me, but I've often thought of enlisting in the Navy. It isn't as profitable, but it's certainly more noble."`
			label salute

> 			`	You arrive at the factory and an older girl, perhaps seventeen, meets the two of you. Gus stands to attention and gives you a well-practiced salute. You politely salute him in return.`

Reviewer 4: ```suggestion
			`	You arrive at the factory, and an older girl, perhaps seventeen, meets the two of you. Gus stands to attention and gives you a well-practiced salute. You politely salute him in return.`
```

			`	"You, on the other hand, own your ship and serve whoever you want. Even if you remain loyal to a group or cause, how and when you act is still at your own discretion. As well, to keep up with the costs of that independent ownership, you have to charge a lot of money for your services. Mercenaries only survive as long as you if they're skilled at both finances and combat."`
				goto questions

mission "War Reactions: Syndicate Orphan"
	landing
	minor
	source
		government "Syndicate"
		not attributes "station"
	to offer
		has "event: post-war reactions begin"
		random < 30
	on offer
		set "pug media coverage"
		conversation
			`	When you exit the secure landing area, you hear sounds of fists falling on skin and a kid yelling for help. Around the corner are three boys hitting a fourth, who just fell on the ground. It's not clear if they're going to stop attacking him.`
			choice
				`	"Break it up, kids."`
				`	(Ignore them and keep walking.)`
					decline
			`	The three attackers turn around, looking furious. They eye your gun and run away. "Thanks. I'm Gus," says the fourth boy, as he slowly rises to his feet. He looks about twelve years old.`

> 			`	It takes Gus a few seconds to recover. He looks worried and asks, "Can you walk me to the factory? I'm already late for work, and I'm afraid they'll attack me again."`

Author: I changed it to "repair shop," and the orphan work is generalized to "industry."

				goto next
			label blew
			`	"I bet you did! I wish I was on one of our warships, bravely defending the human race. The honor, the glory, the mercenary pay. It sounds amazing."`
			label next
			`	"What was it like, fighting the evil Pug?"`

			choice
				`	"Actually, they're not evil. Their 'war' was a ploy to get us to stop fighting among ourselves. A lot of Pug gave their lives willingly to save us from self-destruction."`
				`	"War is hell, kid. I'm not going to glorify it for you."`
					goto hell
				`	"Leading a battle is like chess with guns and engines. It's the most exhilarating feeling I've ever felt."`
					goto chess
				`	"It was an honor to protect the human race from our attackers."`
					goto honor
			`	"That's so insightful! Still, without people like you, the Pug's ploy would never have worked. Thank you for your service, Captain."`
				goto salute
			label hell
			`	"You went through hell for us, eh? That's something to think about. Maybe I should stick to engineering instead of becoming a combat pilot. I dunno."`
				goto salute
			label chess

> 			`	"The Syndicate is training me to maintain and pilot combat ships! One day, I'll fight aliens like you did."`

Reviewer 3: ```suggestion
			`	"The Syndicate is training me to maintain and they're even going to train me to pilot their combat ships when I'm older! One day, I'll fight aliens like you did."`
```

				goto salute
			label honor

> 			`	"Glory, honor, and service. The Syndicate has been training me, but I've often thought of enlisting in the Navy. It isn't as profitable, but it's certainly more noble."`

Reviewer 3: ```suggestion
			`	"Honor, and service. The Syndicate will train boys like me, but I've wondered about joining the Navy. It isn't as profitable, but it sounds more noble."`
```

			label salute
			`	You arrive at the factory, and an older girl, perhaps seventeen, meets the two of you. Gus stands to attention and gives you a well-practiced salute. You politely salute him in return.`

> 			`	As Gus ends his salute, the girl attacks him from behind, putting him into a choke hold. It's clear she's taking extreme care when doing it. "Did you wander out of the safe zone? Was it the spaceport landing area this time? I can see scratches and bruises all over you." Gus replies, "I'm sorry, Sally! I was curious."`

Reviewer 3: If you change the specific nature of his workplace, I'd suggest aligning this to be something like "Did you try and take a shortcut again?" rather than the safe zone reference.

Author: He now works in the childcare district. Entering the nearby spaceport without supervision is exclusively a no-no for repair shop orphans.

				goto questions

mission "War Reactions: Syndicate Orphan"
	landing
	minor
	source
		government "Syndicate"
		not attributes "station"
	to offer
		has "event: post-war reactions begin"
		random < 30
	on offer
		set "pug media coverage"
		conversation
			`	When you exit your ship, you hear sounds of fists falling on skin and a kid yelling for help. Around the corner are three boys hitting a fourth, who just fell on the ground. It's not clear if they're going to stop attacking him.`
			choice
				`	"Break it up, kids."`
				`	(Ignore them and keep walking.)`
					decline
			`	The three attackers turn around, looking furious. They eye your gun and run away. "Thanks. I'm Gus," says the fourth boy, as he slowly rises to his feet. He looks about twelve years old.`
			`	Gus thinks for a second and looks worried. "Can you walk me to the factory? I'm already late for work, and I'm afraid they'll attack me again."`
			choice
				`	"Aren't you a little young to work in a factory?"`

> 				`	"I'm in a hurry. You're on your own."`

Reviewer: Might be interesting for Gus to explain why he's attacked? Just before he says he wants you to walk him back to the factory, or have the player ask it (doesn't flow as well).

Author: Thank you for noticing that. The player is now informed about the reason, if they ask.

					decline
			`	As you start walking, he answers. "No, I'm an orphan." He sees your confusion and explains. "The Syndicate lets some orphans work in factories, in return for better education, good housing, and medical care. Only the cleverest of us get to do it. All of the orphanage factories are safe; we don't use big dangerous equipment. Just ordinary stuff, like hammers, soldering irons, or sewing machines.`

> 			`	"Anyway, you're a captain, right? Were you in the big fights? I was on Maker when I saw some Pug ships fighting human ships on our Syndicate school's big educational viewscreen."`

Reviewer: ```suggestion
			`	"Wait, you're a captain, right? Were you in the big fights? I was on Maker when I saw some Pug ships fighting human ships on our Syndicate school's big educational viewscreen."`
```

			choice
				`	"Yes, I'm Captain <first> <last>."`
					goto me
				`	"I blew up my share of Pug ships."`
					goto blew
				`	"Actually, I led the battle."`
			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You're Captain <first> <last>! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand and you shake it.`
				goto next
			label me
			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You led the attack! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand and you shake it.`

> 			label blew

Reviewer: ```suggestion
				goto next
			label blew
```

			`	"I bet you did! I wish I was on one of our warships, bravely defend the human race. The honor, the glory, the mercenary pay. It sounds amazing."`

> 				goto next

Reviewer: ```suggestion

```

			label next
			`	"What was it like, fighting the evil Pug?"`

			choice
				`	"Actually, they're not evil. Their 'war' was a ploy to get us to stop fighting among ourselves. A lot of Pug gave their lives willingly to save us from self-destruction."`
				`	"War is hell, kid. I'm not going to glorify it for you."`
					goto hell
				`	"Leading a battle is like chess with guns and engines. It's the most exhilarating feeling I've ever felt."`
					goto chess
				`	"I was an honor to protect the human race from our attackers."`
					goto honor
			`	"That's so insightful! Still, without people like you, the Pug's ploy would never have worked. Thank you for your service, captain."`
				goto salute
			label hell

> 			`	"You went through hell for us, eh? That's something to think about. Maybe I should stick to engineering instead of becoming a combat pilot. I donno."`

Reviewer: ```suggestion
			`	"You went through hell for us, eh? That's something to think about. Maybe I should stick to engineering instead of becoming a combat pilot. I dunno."`
```
